src/jit/codegenlegacy.cpp

```diff
@@ -3788,7 +3788,7 @@ void CodeGen::genCondJumpLng(GenTreePtr cond, BasicBlock* jumpTrue, BasicBlock*
 #if CPU_LOAD_STORE_ARCH
         /* Generate the second operand into a register pair */
         // Fix 388442 ARM JitStress WP7
```

Comment:
There was apparently some historical bug associated with this code - did you happen to look into that? It's odd that it would be explicitly passing `false` here (since that's the default).